### ISSUE TITLE
PaddleOCR training: I keep getting an accuracy of 0.00 after multiple epochs.

### ISSUE THREAD

**Issue by aadit2697** [PaddlePaddle/PaddleOCR#9152]
Run command: python tools/train.py -c configs/rec/multi_language/rec_devanagari_lite_train.yml

I am training the OCR for language= Gujrati without a pretrained model. 
this is my config file: https://hastebin.com/share/abesoxafov.yaml

gujrati_dict.txt https://hastebin.com/share/ivateraxiw contains data like:
ક
કા
કિ
કી
કુ
કૂ
કે
કૈ
કો
કૌ
કં
કઃ
ક્ર
ક્રા
.
.
.
train_data/rec_guj_train1.txt contains the  word image path and the word (delim: "||" made the change in simple_dataset.py):

rec/train/1_001.jpg||૫સાર
rec/train/1_002.jpg||અંકાયુ
rec/train/1_003.jpg||અંકિત
rec/train/1_004.jpg||અંકુર
rec/train/1_005.jpg||અંકુરમાં
rec/train/1_006.jpg||અંકુરો
rec/train/1_007.jpg||અંકુશરૂપ
rec/train/1_008.jpg||અંગ
rec/train/1_009.jpg||અંગત

Also, I am attaching my train.log file for your `ref:`
https://hastebin.com/share/wuqorenolo.yaml

Pls note: I have followed the existing issue, but that didn't solve my problem.
Can you help me figure out where am I going wrong?

Thanks in advance! :)

**Comment by aadit2697** [comment]
@WenmuZhou  @dyning  @LDOUBLEV @tink2123 @MissPenguin @Evezerest @littletomatodonkey @andyjpaddle @weisy11 @D-DanielYang @Topdu @sdcb @ZeyuChen @bingooo 

It would be great help, if you could assist me

**Comment by TapendraBaduwal** [comment]
@aadit2697 "Why did you not use devanagari_PP-OCRv3_rec.yml?"

**Comment by aadit2697** [comment]
@TapendraBaduwal  I wanted to try training on a small dataset initially. To check out the training process.
However, let me try the latter out too.

**Comment by TapendraBaduwal** [comment]
@aadit2697 if the dictionary is changed (not recommended), more data set quantities are required.

Can you share these files.
output/rec/
├── best_accuracy.pdopt  
├── best_accuracy.pdparams  
├── best_accuracy.states  
├── config.yml  
├── iter_epoch_3.pdopt  
├── iter_epoch_3.pdparams  
├── iter_epoch_3.states  
├── latest.pdopt  
├── latest.pdparams  
├── latest.states  
└── train.log

**Comment by aadit2697** [comment]
Sure here is the gdrive : https://drive.google.com/drive/folders/1DnjoZYHlivcHmNQrhLfrb8jn86OXaVza?usp=sharing

Pls note: it does not contain since i didnt let it run completely as no improvement in acc was observed
├── best_accuracy.pdopt
├── best_accuracy.pdparams
├── best_accuracy.states

P.S : bare with me as it took some time to upload files. pc is slow due to the ongoing training process

**Comment by TapendraBaduwal** [comment]
@WenmuZhou @dyning @LDOUBLEV @tink2123 @MissPenguin @Evezerest @littletomatodonkey @andyjpaddle @weisy11 @D-DanielYang @Topdu @sdcb @ZeyuChen @bingooo

**Comment by TapendraBaduwal** [comment]
@aadit2697  How much data do you have?
eval_batch_step: [0, 2000]
The above parameters indicate that it is evaluated once after 2000 iterations. If you have very little data and the number of iterations does not reach 2000, no evaluation will be performed and the best_accuracy model will not be saved.
So set eval_batch_step: [0, 20] and infer with best_accuracy model.

Can you share the dataset?
